components/shared/embedder/lib.rs

```diff
@@ -223,6 +225,18 @@ pub enum AllowOrDeny {
     Deny,
 }
 
+/// Data about a `WebView` of `<iframe>` viewport: its size and also the
```

Comment:
"Data about a `WebView` of `<iframe>` viewport" is hard to parse for me. Could this be rephrased?